src/main/java/bio/terra/service/filedata/azure/util/AzureConstants.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.service.filedata.azure.util;
+
+public class AzureConstants {
```

Comment:
If you refactor out the common `ManagementException` handling, you could put the method in this class.

Reply:
I mentioned above, sadly, I don't think the common managementexception stuff is common enough to pull into a separate method